### ISSUE TITLE
Migrate from build dependency on jcenter() 

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-app-android#3264]
## Avoid duplicates
* [X] This enhancement request has not already been raised before
* [X] Enhancement request is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)
* [ ] If you are proposing a new feature, please do so in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)

## Current Implementation

`build.gradle` uses `jcenter()` in `repositories` and `allprojects` 

Android Studio 4.2.1 produces a new warning, which was not given in Android Studio 4.1.x and earlier versions.

```
"Please remove usages of `jcenter()` Maven repository from your build scripts and migrate your build to other Maven repositories.
This repository is deprecated and it will be shut down in the future.
See http://developer.android.com/r/tools/jcenter-end-of-service for more information.
Currently detected usages in: root project 'Corona-Warn', project ':Corona-Warn-App', project ':Server-Protocol-Buffer'"


```

## Suggested Enhancement

- As first step add `mavenCentral()` to `build.gradle` before each occurrence of  `jcenter()`
- later, once it is certain that `jcenter()` is no longer needed, remove references to `jcenter()`

The Google Exposure Notification reference design
https://github.com/google/exposure-notifications-android/commit/ab04ef755cde964a43aa94de8b9d86084d0b4f6f
has already added `mavenCentral()`

## Expected Benefits

Change remediates deprecated dependency and improves readability of build log.

---
Internal Tracking ID: [EXPOSUREAPP-7437](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7437)

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks.


---
Internal Tracking ID: [EXPOSUREAPP-7437](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7437)
Corona-Warn-App Open Source Team

**Comment by dsarkar** [comment]
FYI PR https://github.com/corona-warn-app/cwa-app-android/pull/3272

**Comment by MikeMcC399** [comment]
This issue is fixed and verifiable in the [release/2.4.x](https://github.com/corona-warn-app/cwa-app-android/tree/release/2.4.x) release branch by running `gradle help` in a terminal window.

Since it only affects developers, not end-users of the app, and it is already fixed, there is no need to wait for the release of version 2.4 before closing. So I am closing this issue here.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks.
